core/controlplane/config/templates/stack-template.json

```diff
@@ -461,6 +488,11 @@
             "Key": "Name",
             "PropagateAtLaunch": "true",
             "Value": "{{$.ClusterName}}-{{$.StackName}}-kube-aws-etcd-{{$etcdIndex}}"
+          },
+          {
+            "Key": "kube-aws:role",
```

Comment:
can't it use Name tag above? it kind find its own tag and then strip index to find other etcd instances